### PR TITLE
Added nextjs example + fixed create-react-app example

### PR DESCRIPTION
**Pull request by chrisvanmook**
## Amsterdam styled components pull request

Before opening a pull request, please ensure:

- [ ] You've added or updated the README.md of the component
- [ ] You have been following the guidelines, written in the [README](../blob/master/docs/CONTRIBUTING.md#user-content-conventions-and-rules) file
- [ ] Your code has the necessary tests written
- [ ] You've exported the component in [index.ts](./packages/asc-ui/src/index.ts)
- [ ] You have updated the [CHANGELOG.md unreleased sections](../blob/master/CHANGELOG.md#muser-content-unreleased)

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

### CODE REVIEW

File: examples/nextjs/.babelrc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["next/babel"],
+  "plugins": [["styled-components", { "ssr": true }]]
```
Comment (jonkoops) [inline examples/nextjs/.babelrc:3]: I would also add `"pure": true` to the config here so more dead code can be eliminated. See https://styled-components.com/docs/tooling#dead-code-elimination

File: examples/nextjs/pages/index.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import { Close } from '@datapunt/asc-assets'
+import GlobalStyle from '@datapunt/asc-ui/lib/components/GlobalStyle'
+import Header from '@datapunt/asc-ui/lib/components/Header'
+import Button from '@datapunt/asc-ui/lib/components/Button'
+import Typography from '@datapunt/asc-ui/lib/components/Typography'
+
+export default function Home() {
+  return (
+    <>
+      <GlobalStyle />
```
Comment (jonkoops) [inline examples/nextjs/pages/index.jsx:11]: Should we not be including `GlobalStyle` in the `App` component?

File: examples/nextjs/pages/_app.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import App from 'next/app'
+import React from 'react'
+import ThemeProvider from '@datapunt/asc-ui/lib/theme/ThemeProvider'
```
Comment (jonkoops) [inline examples/nextjs/pages/_app.js:3]: Should we not be importing this from the index of `@datapunt/asc-ui` instead?

File: examples/nextjs/pages/index.jsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { Close } from '@datapunt/asc-assets'
+import GlobalStyle from '@datapunt/asc-ui/lib/components/GlobalStyle'
```
Comment (jonkoops) [inline examples/nextjs/pages/index.jsx:3]: Should we not be importing this from the index of @datapunt/asc-ui instead

File: examples/nextjs/public/zeit.svg [hunk 5]
```diff
@@ -0,0 +1,10 @@
+<svg width="82" height="16" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (jonkoops) [inline examples/nextjs/public/zeit.svg:1]: I think we can get rid of this file

File: examples/nextjs/pages/index.jsx (hunk 2 above)
Comment (chrisvanmook) [reply examples/nextjs/pages/index.jsx:11]: will do!

File: examples/nextjs/pages/_app.js (hunk 3 above)
Comment (chrisvanmook) [reply examples/nextjs/pages/_app.js:3]: Yes, we can do that now!

**Pull request merged**